### PR TITLE
Reformat postgres cgroup v2 fix #201

### LINKED ISSUE #201: [postgres] alter_system.py does not use memory and cpu limits
Read this comment for update:
https://github.com/hzi-braunschweig/SORMAS-Docker/issues/201#issuecomment-829046545

---
Old description:

The return values of `get_mem` and `get_cpu` are unused:
```python
def get_tuning_values(config, filename):
    mem = get_mem()
    cpu = get_cpu()
```
and therefore the functions should be removed

### Why is this a problem anyway?
Well, the Linux Kernel. Apparently my last OS update switched to [cgroup v2](https://www.kernel.org/doc/html/latest/admin-guide/cgroup-v2.html) which has a completely different layout of `/sys/fs/cgroup/` and files like e.g., `memory/memory.limit_in_bytes` are gone, renamed, or replaced causing `alter_system.py` to fail.

### How to resolve the issue
1. Delete `get_mem` and `get_cpu` as these functions are not used
2. Use the return values but then we need to future proof the logic

### PR DESCRIPTION
**Pull request by norpol**
Kind of an alternative suggestion for the detection of cgroup limits, but actually just cosmetic so happy to drop it again.

### CODE REVIEW

File: postgres/alter_system.py [hunk 1]
```diff
@@ -59,55 +58,44 @@ def human_to_int(value):
...
+      # cgroup v1
+      try:
+          mem = open("/sys/fs/cgroup/memory/memory.limit_in_bytes").read().rstrip()
+      # no cgroup memory limits configured, assuming max
+      except IOError:
+          mem = "max"
+
+  # max in cgroup v2, -1 in v1
+  # if max or cgroup limit mem bigger than sysmem, use sysmem, else use cgroup mem limit
+  return sysmem if mem == "max" or int(mem) > sysmem else int(mem)
```
Comment (norpol) [inline postgres/alter_system.py:79]: Ups a bit of a copy paste error, I'm a bit confused what the fallback for `/sys/fs/cgroup/memory/memory.limit_in_byte` is, if it is `-1` this is of course incorrect.

Comment (TomHutter) [reply postgres/alter_system.py:79]: I am not sure, if you resolved your confusion yourself. In cgroups v1 you get -1 if there is no memory limit, in v2 it's "max". It looks to me, that result -1 (no limit in v1) is currently not covered. So why not use the same pattern in line 79 as you did in line 98:

`return sysmem if mem in ("max", -1) or int(mem) > sysmem else int(mem)`

Comment (norpol) [reply postgres/alter_system.py:79]: Cool! Happy this change is appreciated.

I basically did the same as it was in your code.

https://github.com/hzi-braunschweig/SORMAS-Docker/pull/221/commits/f8648c526f60933f303fdc8292008629ed5fe254#diff-d55335daa9b1401ee6fd0c285186934876f5c891a9b157f8e3e28f4ba28d78a1R83-R88

I guess good that I had a look, I'll adjust the commit accordingly.

Comment (norpol) [reply postgres/alter_system.py]: Ah found another issue it should be `'-1'` not `-1` of course. I've updated the script

**Review by kiney — APPROVED** [review]
just tested the change, works as expected

**Review by asohh — APPROVED** [review]

**Pull request merged**